src/page.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComponentChildren, VNode } from "preact";
+import Header from "./header";
+
+interface PageProps {
+  path: string;
+  children?: ComponentChildren;
+  title: string;
+}
+
+function Page({ path, children, title }: PageProps): VNode {
+  return (
+    <>
+      <Header path={path} />
+      <div className="grid-container">
```

Comment:
I *think* we want the grid container to not include the header? so I stuck it here